File: css/money/ch09/Money.Tests/DollarTest.cs
using NUnit.Framework;
using TheSoftwareGorilla.TDD.Money;

namespace TheSoftwareGorilla.TDD.Money.Tests;

public class DollarTests
{
    [SetUp]
    public void Setup()
    {
    }


    [Test]
    public void TestMultiplication()
    {
        Money five = Money.Dollar(5);
        Assert.That(Money.Dollar(10), Is.EqualTo(five.Times(2)));
        Assert.That(Money.Dollar(15), Is.EqualTo(five.Times(3)));
    }

}

File: java/ch09/src/test/java/com/thesoftwaregorilla/tdd/money/DollarTest.java
package com.thesoftwaregorilla.tdd.money;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

public class DollarTest {


    @Test
    public void testMultiplication() {
        Money five = Money.dollar(5);
        assertEquals(Money.dollar(10), five.times(2));
        assertEquals(Money.dollar(15), five.times(3));
    }

}
